recipes/cpp-half/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cpp-half" %}
+{% set version = "2.1.0" %}
+{% set sha256 = "b287a89d37a1cbf305113c152ee2ba9a58802a12233569909c4ce068d78e31b7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/davidbrochart/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+  host: []  # Empty host dependency section
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/half_float/half.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/halfFloat/halfFloatConfig.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\half_float\half.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\halfFloatConfig.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: http://half.sourceforge.net
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: Half-precision floating-point library
+  description: |
+    This is a C++ header-only library to provide an IEEE 754 conformant 16-bit
+    half-precision floating-point type along with corresponding arithmetic
+    operators, type conversions and common mathematical functions. It aims for
+    both efficiency and ease of use, trying to accurately mimic the behaviour
+    of the built-in floating-point types at the best performance possible.
+
+extra:
+  recipe-maintainers:
+    - davidbrochart
+    - SylvainCorlay
+    - JohanMabille
```

Comment:
CC @SylvainCorlay @JohanMabille - are you willing to be maintainers?

Reply:
yes